### ISSUE TITLE
[Feature request] Custom wallpapers

### ISSUE THREAD

**Issue by imshinyu** [fork-maintainers/iceraven-browser#697]
this may be hard but @akliuxingyuan made the core of how it works so it should be possible

**Comment by mrxyzl** [comment]
I totally agree, Mozilla already make the core for wallpaper changing, now we just need to poit it to external source

**Comment by imshinyu** [comment]
then it should not be that hard to implement it fully

**Comment by sankhababu** [comment]
Android Waterfox did this, though buggy.

**Comment by Fytdyh** [comment]
Do you have any idea if this small group of developers or volunteers has a patreon? While I can't program, I can help buy them a monthly coffee. Little by little we can make this app good
